src/screens/TopicPageScreen/index.js

```diff
@@ -460,6 +439,10 @@ const TopicPageScreen = (props) => {
           onScroll={handleScroll}
           scrollEventThrottle={16}
           showSearchBar={true}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are adding two new functions to the `onMomentumScrollEnd` event: `onWillSendViewPostTime` and `fetchNextFeeds`. It's important to ensure that these functions handle errors properly and do not block the UI thread. If these functions perform heavy computations or network requests, consider moving them to a background thread or making them asynchronous to prevent blocking the UI. However, without seeing the implementation of these functions, I can't provide specific suggestions. Please review these functions for potential performance issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->